components/molecules/TeamMembersConfig/team-members-config.tsx

```diff
@@ -1,30 +1,89 @@
 import Button from "components/atoms/Button/button";
 import Search from "components/atoms/Search/search";
 import TeamMemberRow from "../TeamMemberRow/team-member-row";
+import { useState } from "react";
+import { validateEmail } from "lib/utils/validate-email";
+import { useToast } from "lib/hooks/useToast";
 
 interface TeamMembersConfigProps {
   className?: string;
   members: TeamMemberData[];
+  onAddMember: Function;
+  onDeleteMember: Function;
+  onUpdateMember: Function;
```

Comment:
Same for these. Type is too generic and doesn't represent the input or output of the function.